### PR TITLE
Added option to configure module to be silent (console.log supression)

### PR DESCRIPTION
**Pull request by AndreKirchner**

### CODE REVIEW

**Comment by puneet0191** [comment]
@AndreKirchner thanks for your contribution, but I think you have not made changes on the latest code, please fetch the latest code from master and send PR, also providing a background for this change would be a great help, please

File: readme.md [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
 {
-   "helpers": {
-     "SauceHelper": {
-      "require": "codeceptjs-saucehelper"
-    },
-   }
+  "helpers": {
+    "SauceHelper": {
+      "require": "codeceptjs-saucehelper",
+      "user"   : "SAUCELABS_USERNAME",
```
Comment (puneet0191) [inline readme.md:20]: We removed it because it can be used from Webdriver Helper config, the same for key

**Review by puneet0191 — CHANGES_REQUESTED** [review]

**Comment by AndreKirchner** [comment]
I can see the issue now. You have not updated the version in package.json with your latest updates - therefore I have been working on an older version. Please update the version number - npm caches might be corrupted due to multiple versions under the same number. Thanks!

**Comment by AndreKirchner** [comment]
I also think npm registry is not able to determine your changes due to the missing update on the version number -> There will be old versions installed, when using npm i regulary.

**Comment by puneet0191** [comment]
@AndreKirchner Fixed, thanks a lot! :+1:

**Pull request closed without merging**